packages/nextjs/components/scaffold-eth/Contract/InheritanceTip.tsx

```diff
@@ -0,0 +1,14 @@
+export const InheritanceTip = ({ inheritedFrom }: { inheritedFrom?: string }) => {
+  return (
+    <>
+      {inheritedFrom && (
+        <span
+          className="tooltip tooltip-top tooltip-secondary px-2 before:content-[attr(data-tip)]"
```

Comment:
Maybe add `cursor-help`? I think I saw it in some of the videos.

Reply:
Yup, I removed it for when it was possible to copy from the custom tooltip. I will put it back now.

Reply:
Fixed here: https://github.com/FilipHarald/scaffold-eth-2-sandbox/commit/92aa41f13f354fb982da16e4d4c578ce8b086cad